uber/models.py

```diff
@@ -367,13 +376,14 @@ def __getattr__(self, name):
             return suffixed
 
         try:
-            [multi] = [col for col in self.__table__.columns if isinstance(col.type, MultiChoice)]
             choice = getattr(c, name)
-            assert choice in [val for val, desc in multi.type.choices]
+            if len(self.multichoice_columns) == 1:
+                multi = self.multichoice_columns[0]
+                if choice in multi.type.choices_dict:
+                    return choice in getattr(self, multi.name + '_ints')
         except:
```

Comment:
Yeah, I don't really like this either. I was following the style of the old code. Maybe I should change this to your suggestion.